homeassistant/components/roborock/__init__.py

```diff
@@ -37,31 +38,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     home_data = await api_client.get_home_data(user_data)
     _LOGGER.debug("Got home data %s", home_data)
 
-    devices_info: dict[str, RoborockDeviceInfo] = {}
+    mqtt_devices_info: dict[str, RoborockDeviceInfo] = {}
     devices: list[HomeDataDevice] = home_data.devices + home_data.received_devices
     for device in devices:
-        devices_info[device.duid] = RoborockDeviceInfo(device)
+        mqtt_devices_info[device.duid] = RoborockDeviceInfo(device)
```

Comment:
I think you can also shorten to something like this:
```
mqtt_devices_info = {
    device.duid: RoborockDeviceInfo(device)
    for device in devices
}
```

or even inline further as an argument to `RoborockMqttClient`